sidebars.js

```diff
@@ -512,6 +516,7 @@ module.exports = {
             type: 'doc',
             id: 'exchange-operators/rosetta/index',
```

Comment:
see my earlier comments about the section levels... I strongly recommend moving the Rosetta API in one level so it's not unnecessarily nested